openqa-label-known-issues

```diff
@@ -51,18 +51,26 @@ handle_unreachable() {
         return 1
     fi
     # resorting to downloading the job details page instead of the
-    # log, overwrites $out
-    if ! curl "${curl_args[@]}" -s "$testurl" -o "$out"; then
+    # log
+    if ! curl "${curl_args[@]}" -s "$testurl" -o "${html_out}"; then
         echoerr "'$testurl' can be reached but not downloaded, bailing out"
         curl "${curl_args[@]}" "$testurl"
         exit 2
     fi
+    local timeago=$(grep timeago "$html_out" | hxselect -s '\n' -c '.timeago::attr(title)')
 
-    if hxnormalize -x "$out" | hxselect -s '\n' -c '.links_a .resborder' | grep -qPzo '(?s)Gru job failed.*connection error.*Inactivity timeout'; then
+    if hxnormalize -x "${html_out}" | hxselect -s '\n' -c '.links_a .resborder' | grep -qPzo '(?s)Gru job failed.*connection error.*Inactivity timeout'; then
         "${client_call[@]}" -X POST jobs/"$id"/comments text='poo#62456 test incompletes after failing in GRU download task on "Inactivity timeout" with no logs'
         "${client_call[@]}" -X POST jobs/"$id"/restart
         return 1
     fi
+    # Checking timestamp given by job details page
+    if [[ $(date -uIs -d '-14days') > $timeago ]]; then
+        # if the page is there but not even an autoinst-log.txt exists
+        # then the job might be too old and logs are already deleted.
+        echoerr "'$testurl' job#${id} without autoinst-log.txt older than 14 days. Do not label"
+        return
```

Comment:
```suggestion
        return 1
```
Like I said in the meeting, you have to return with non zero here now of course, so that the later check when calling handle_unreachable will return

Reply:
@perlpunk as I didnt follow the exact logic, this request havent applied. Can we close and open once again if you find it neccessary?